src/Models/Services/StoryUtilities.tsx

```diff
@@ -132,7 +132,14 @@ export const selectReactSelectOption = (
     fireEvent.mouseDown(
         document.getElementsByClassName(`${className}__control`)[0]
     );
-    fireEvent.click(
-        document.getElementsByClassName(`${className}__option`)[optionIndex]
+    // Wait for callout to pop-up
+    setTimeout(
```

Comment:
this might work locally, but it will break when we get back to chromatic tests because the test will end and take the picture before this timeout fires.

Reply:
I typically do an `await sleep(1)` and then it will give the render loop a chance to complete and then comes back here to execute. It just pushes your function execution to the bottom of the javascript execution stack.